src/command_modules/azure-cli-kusto/azure/cli/command_modules/kusto/_params.py

```diff
@@ -6,17 +6,19 @@
 # pylint: disable=line-too-long
 from knack.arguments import CLIArgumentType
 
-from azure.cli.core.commands.parameters import (name_type)
-from azure.mgmt.kusto.models import AzureSkuName
-from azure.cli.core.commands.parameters import (get_enum_type)
+from azure.cli.core.commands.parameters import (name_type, get_enum_type)
+
+from .custom import (
+    AzureSkuName
+)
 
 
 def load_arguments(self, _):
 
     # Kusto clusters
     sku_arg_type = CLIArgumentType(help='The name of the sku.',
                                    arg_type=get_enum_type(AzureSkuName))
-    time_format_explenation = 'Time format is DDDD:HH:MM:SS (for example, 10 years would be 3650:00:00:00).'
+    time_format_explenation = 'Duration in ISO8601 format (for example, 100 days would be P100D).'
```

Comment:
There's no bump in the SDK version here. What format does the service actually require?

Reply:
There is a bump in the SDK, this PR uses version 2019-01-21, while the current code uses version 2018-09-07-preview.
In the new SDK, the type of the duration is ISO8601